Packages/Sites/Neos.NeosIo/Configuration/NodeTypes.Stage.yaml

```diff
@@ -20,6 +20,7 @@
           'TYPO3.Neos.NodeTypes:TextWithImage': TRUE
           'TYPO3.Neos.NodeTypes:Headline': TRUE
           'Neos.NeosIo:Button': TRUE
+          'Neos.NeosIo:ChoiceButtons': TRUE
```

Comment:
Code style : We discussed in our call with the other website people to use lowercase true/false in yaml as standard